n2y/main.py

```diff
@@ -95,11 +99,14 @@ def export_markdown(client, raw_rows, options):
             if pandoc_output:
                 markdown = pandoc.write(pandoc_output, format='gfm+tex_math_dollars') \
                     .replace('\r\n', '\n')  # Deal with Windows line endings
-                # reformat equations to have one backslash instead of 2
+
+                # create target path if it doesn't exist
```

Comment:
I don't think this comment is necessary since it's pretty clear from `os.makedirs(options.target, exist_ok=True)` what the line does